components/LinkGrid/src/index.scss

```diff
@@ -0,0 +1,183 @@
+.denhaag-link-grid {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  overflow-x: hidden;
+  background-color: var(--denhaag-link-grid-panel-background-color);
+  padding-block-start: var(--denhaag-link-grid-panel-padding-sm);
+  padding-block-end: var(--denhaag-link-grid-panel-padding-sm);
+
+  @media (min-width: 1024px) {
```

Comment:
Om te voorkomen dat er teveel @media queries komen, lijkt mij dit beter helemaal onderin te doen.
Dus 1 @media breakpoint voor meerdere selectors:

```
@media (min-width: 768px) {
.selector-1 { ... }
.selector-2 { ... }
.selector-3 { ... }
}

@media (min-width: 1024px) {
.selector-1 { ... }
.selector-2 { ... }
.selector-3 { ... }
}
```

Reply:
Hi @robbieatacato en @paulAcato 

Dit is vrij persoonlijk, maar ik vind het fijner om juist de breakpoints binnen de class te hebben. Als je dan bezig bent met een bepaalde class of component, kun je meteen alle breakpoints checken. Anders moet ik door de hele css gaan zoeken waar die classes nog meer gebruikt worden. Als dit een GDH requirement is kan ik het wel aanpassen, maar ik ben benieuwd wat jullie hiervan denken?